docs/basics/functions.md

```diff
@@ -665,6 +669,110 @@ Which would return the following:
 "metrics": []
 ```
 
+### sample_include_filter
+
+| Applies to | Description |
+| :--------- | :---------- |
+| API | Only creates the sample if both a key and value is found in the sample.|
+|| In the absence of `sample_include_filter`, it defaults to include all.|
+|| If a sample is included by the `sample_include_filter` filter, the `sample_filter` and `sample_exclude_filter` will be evaluated next.|
+|| If a sample is NOT included by the `sample_include_filter` filter, the `sample_filter` and `sample_exclude_filter` will be skiped.|
```

Comment:
Again, I don't think this part is necessary. I take that if the include filter is not used, all the others will be evaluated?

Reply:
will be skiped > are skipped